content/llm/speaking-bots.md

```
@@ -542,19 +538,19 @@ Deploy AI-powered speaking agents in video meetings
 
 
 <Callout type="info">
-  We provide detailed documentation optimized for both human readers and AI assistants. For more information about our LLM integration, see
-  [LLMs](../llms/available).
+  We provide optimized documentation for both LLMs and recent MCP server updates. For more on our LLM integration, 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)